examples/kprobe/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/cilium/ebpf"
+
+	"golang.org/x/sys/unix"
+)
+
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cflags "$BPF_CFLAGS" -cc clang-10 KProbeExample ./bpf/kprobe_example.c
```

Comment:
$BPF_CFLAGS is probably undefined here, so there is not much point in passing the variable. We currently don't test against clang-10, so clang-11 is probably more appropriate.